content/docs/welcome-guide/setup/requirements.md

```diff
@@ -26,34 +30,83 @@ Minimum hardware requirements for development include the following:
 RAM and free disk space requirements are dependent on the options that you select when configuring your project in O3DE.
 {{< /note >}}
 
-## Software prerequisites
+## Software prerequisites and configuration
```

Comment:
There's a link into this subsection from the Setup Troubleshooting topic (`/welcome-guide/setup/troubleshooting.md`) that would be broken with this change. One way you could prevent this broken link is by defining a fixed relative link here, by appending this to the line: `{#software-prerequisites}`.

Reply:
Of course, then you'd also need to adjust your updated link in the first paragraph on this page.